backstop/tests/date-picker.html

```diff
@@ -1504,74 +1504,75 @@ <h2>Date & time</h2>
             </div>
           </div>
         </div>
+        <div class="iui-time-picker">
```

Comment:
```suggestion
        <div class="iui-time-picker" role="dialog" aria-modal="true">
```
Copying the accessibility from lines 32-35.